index.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
```

Comment:
## Improve the general layout
- [ ] We currently do have all the sections of the page covering the entire view width of the page. This shouldn't be so. We do need to, however, have only the header, the three main articles and the footer covering the full width of the page. the `Latest funding` section ought to be centred properly. Kindly fix this.

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://1v4n4.github.io/Responsive-web-page/)|
|:--:|:--:|
|![tnw](https://user-images.githubusercontent.com/22345961/113048559-59629480-91ab-11eb-841f-28a2729052af.png)|![tnw-clone](https://user-images.githubusercontent.com/22345961/113048599-64b5c000-91ab-11eb-8e87-c2870a74f88c.png)|